src/bindings/python/BUILDING.md

```diff
@@ -1,88 +1,145 @@
-    cd openvino/build
-
-    cmake .. \
-        -DENABLE_INTEL_GPU=OFF \
-        -DENABLE_OPENCV=OFF \
-        -DENABLE_INTEL_MYRIAD_COMMON=OFF \
-        -DENABLE_PYTHON=ON \
-        -DENABLE_OV_ONNX_FRONTEND=ON \
-        -DCMAKE_INSTALL_PREFIX="${OPENVINO_BASEDIR}/openvino_dist"
-
-    make -j 4
-    make install
-
-The Python module is installed in the `${OPENVINO_BASEDIR}/openvino_dist/python/python<version>/` folder.
+the building of a few modules and plugins by setting the `ENABLE_*` flag to `OFF`.
+
+In order to build the OpenVINO™ Python APIs set the `ENABLE_PYTHON` flag to `ON`.
+
+Note the `CMAKE_INSTALL_PREFIX`, which defaults to `/usr/local/` if not set.
+
+```bash
+cd ${OV_WORKSPACE}
+git clone --recursive https://github.com/openvinotoolkit/openvino.git
+cd openvino
+git submodule update --init
+
+### You may also want to install OpenVINO™ dependencies with:
+# ./install_build_dependencies.sh
+
+mkdir build && cd build
+
+cmake .. \
+-DCMAKE_BUILD_TYPE=Release \
+-DENABLE_INTEL_CPU=ON \
+-DENABLE_INTEL_GNA=OFF \
+-DENABLE_INTEL_GPU=OFF \
+-DENABLE_INTEL_MYRIAD_COMMON=OFF \
+-DENABLE_INTEL_MYRIAD=OFF \
+-DENABLE_HETERO=OFF \
+-DENABLE_MULTI=OFF \
+-DENABLE_AUTO=ON \
+-DENABLE_TEMPLATE=ON \
+-DENABLE_PYTHON=ON \
+-DENABLE_OV_IR_FRONTEND=ON \
+-DENABLE_OV_ONNX_FRONTEND=ON \
+-DCMAKE_INSTALL_PREFIX="${OV_WORKSPACE}/openvino_dist"
+
+make -j 4 install
+```
+
+The Python module is installed in the folder:
+
+    ${OV_WORKSPACE}/openvino_dist/python/python<version>/
+
+You may also find it in:
+
+    ${OV_WORKSPACE}/openvino/bin/intel64/[BUILD_TYPE]/python_api/python<version>/
+
+If you would like to use a specific version of Python, or use a virtual environment, you can add the `PYTHON_EXECUTABLE` variable to your CMake command line. For example:
+
+```bash
+-DPYTHON_EXECUTABLE=/path/to/venv/bin/python
+# or while in virtual environment
+-DPYTHON_EXECUTABLE=`which python`
+```
+
 Set up the OpenVINO™ environment in order to add the module path to `PYTHONPATH`:
 
-    source ${OPENVINO_BASEDIR}/openvino_dist/setupvars.sh
+```bash
+source ${OV_WORKSPACE}/openvino_dist/setupvars.sh
+```
 
-If you would like to use a specific version of Python, or use a virtual environment, you can set the `PYTHON_EXECUTABLE`
-variable. For example:
+If you want more control over enviroment variables, to simply enable OpenVINO™ Python API, export these variables:
 
+```bash
+export LD_LIBRARY_PATH=${OV_WORKSPACE}/openvino_dist/runtime/lib/intel64/
+export PYTHONPATH=${OV_WORKSPACE}/openvino_dist/python/python<version>/
 ```
--DPYTHON_EXECUTABLE=/path/to/venv/bin/python
--DPYTHON_EXECUTABLE=$(which python3.8)
+
+### Rebuild OpenVINO™ project
+
+To rebuild project, simply navigate to build folder and re-run `make` command:
+```bash
+cd ${OV_WORKSPACE}/openvino/build
+
+make -j 4 install
 ```
+*Note: If changes are adding new compilation units, files or change CMake scripts, there is a need to remove the exisiting build and rebuild it from scratch!*
```

Comment:
Would the info that all `bin`, `build` and `openvino_dist` directories should be removed be useful?

Reply:
It might be... let me add it in new PR if needed.